std/special/build_runner.zig

```diff
@@ -202,7 +202,7 @@ fn usage(builder: *Builder, already_ran_build: bool, out_stream: var) !void {
 
 fn usageAndErr(builder: *Builder, already_ran_build: bool, out_stream: var) anyerror {
```

Comment:
I think this is a good change, but now we should change the return type of `usageAndErr` to `noreturn`

Reply:
`noreturn` gave me `zig/build/lib/zig/std/special/build_runner.zig:44:11: error: unreachable code defer builder.deinit();`

Maybe use `void` instead?

Reply:
Ah. That's problematic. I created #2198 for that, and yeah void in the meantime is fine.